src/Meshes/Builders/shapeBuilder.ts

```diff
@@ -27,7 +29,7 @@ import { Path3D } from "../../Maths/math.path";
  */
 export function ExtrudeShape(
     name: string,
-    options: { shape: Vector3[]; path: Vector3[]; scale?: number; rotation?: number; cap?: number; updatable?: boolean; sideOrientation?: number; frontUVs?: Vector4; backUVs?: Vector4; instance?: Mesh; invertUV?: boolean },
+    options: { shape: Vector3[]; path: Vector3[]; scale?: number; rotation?: number; closeShape?: boolean; closePath?: boolean; cap?: number; updatable?: boolean; sideOrientation?: number; frontUVs?: Vector4; backUVs?: Vector4; instance?: Mesh; invertUV?: boolean },

```

Comment:
This is unfortunately a breaking change, could both params be added at the end of the function ?

Reply:
This change does not break existing code they only come into action when specifically set, defaults match existing code. Order does not matter as they are properties of the options parameter object not a parameter of the function.